int8 intensity
bool ping_rcvd
char[8] fault